shared/src/vyxal/Context.scala

```diff
@@ -105,7 +107,9 @@ class Context private (
     else stack.toList.reverse ::: globals.inputs.peek(n - stack.length)
 
   /** Push items onto the stack. The first argument will be pushed first. */
-  def push(items: VAny*): Unit =
+  def push(itms: VAny*): Unit =
+    val items = itms.filterNot(_.isInstanceOf[VNull])
+    if items.isEmpty || items.isInstanceOf[VNull] then return
```

Comment:
`items.isInstanceOf[VNull]` will never be true. `items` is a `Seq[VAny]`. You're already filtering out nulls when creating `items`

```suggestion
    if items.isEmpty then return
```

Reply:
Honestly you don't even need to bother with this early return behavior, unless you have reason to believe this optimization will bring about a noticeable performance boost. I'm fine either way though